cfgov/unprocessed/css/pages/careers.less

```diff
@@ -3,10 +3,22 @@
    Career pages at /careers/.
    ========================================================================== */
 
-// TODO: Consolidate site-wide media_image responsive sizes into one class.
-.students-and-graduates .media_image {
-    width: 130px;
-    .respond-to-min(@bp-med-min, {
-        width: 150px;
-    });
+.t-careers {
```

Comment:
Sorry, I should have noted. Template. I figured we do that with organisms, molecules, atoms.